pages/lazer/integrate-as-consumer/svm.mdx

```diff
@@ -14,19 +14,34 @@ Integrating with Pyth Lazer in smart contracts as a consumer is a three-step pro
 
 ### Use Pyth Lazer SDK into smart contracts
 
-Pyth Lazer provides a [Rust SDK](https://github.com/pyth-network/pyth-crosschain/tree/main/lazer/sdk/rust), which allows consumers to parse the price updates.
+Pyth Lazer provides a [Solana SDK](https://docs.rs/pyth-lazer-solana-contract/latest/pyth_lazer_solana_contract/), which allows consumers to parse and verify the price updates on Solana-compatible chains.
 
-Add the following to your `Cargo.toml` file:
+To start, add the following to your `Cargo.toml` file:
 
 ```toml copy
 [dependencies]
-pyth-lazer-sdk = 0.1.0
+pyth-lazer-solana-contract = { version = "0.2.0", features = ["no-entrypoint"] }
```

Comment:
let's change it to `x.y.z` and ask people to look it up from crate.rs/path/to/crate